src/ontology/metadata/omim.yml

```diff
@@ -3,6 +3,7 @@ label: Online Mendelian Inheritance in Man
 prefix_map:
    OMIM: https://omim.org/entry/
    OMIMPS: https://www.omim.org/phenotypicSeries/PS
+   OMIMPS2: https://omim:org/phenotypicSeries/
```

Comment:
This addition makes no sense, the URI is not valid and way too short as well. Why was this added?

Reply:
Actually that's probably an embarrassing mistake, sorry for wasting your time. Removed.

1st, it wasn't supposed to be `omim:org`. 2nd, I thought I had an example but didn't write it down, but I think my error was that I in passing read `/PS` as `/PS/`, and I saw valid resolvable URLs where the last `/` was after `phenotypicSeries/`. If you don't understand, never mind, I just removed this change.